tests/baselines/reference/thisTypeInAccessorsNegative.errors.txt

```diff
@@ -22,9 +21,6 @@ tests/cases/conformance/types/thisType/thisTypeInAccessorsNegative.ts(16,22): er
     }
     const contextual: Foo = {

```

Comment:
Can you move this test case to `thisTypeInAccessors.ts` so that we can see the types?